flytekit/types/structured/structured_dataset.py

```diff
@@ -647,6 +649,8 @@ def to_html(self, ctx: FlyteContext, python_val: typing.Any, expected_python_typ
             return pd.DataFrame(df).describe().to_html()
         elif importlib.util.find_spec("pyspark") is not None and isinstance(df, pyspark.sql.DataFrame):
             return pd.DataFrame(df.schema, columns=["StructField"]).to_html()
+        elif importlib.util.find_spec("polars") is not None and isinstance(df, pl.DataFrame):
+            return df.to_pandas().describe().to_html()
```

Comment:
This is pretty inefficient as it doubles the memory usage (making a copy of the dataframe). Polars itself doesn't have html rendering so I guess we need to write an html renderer for it?

Reply:
Ah I see for Spark we only render the schema, should do the same.